### PR TITLE
Update recommended schema version in config reference

### PR DESCRIPTION
**Pull request by adeverteuil**

### CODE REVIEW

**Review by JStickler — APPROVED** [review]
[docs team] LGTM

File: docs/sources/configuration/_index.md [hunk 1]
```diff
@@ -3649,7 +3649,7 @@ The `period_config` block configures what index schemas should be used for from
 # If omitted, defaults to the same value as store.
 [object_store: <string> | default = ""]
 
-# The schema version to use, current recommended schema is v11.
+# The schema version to use, current recommended schema is v12.
```
Comment (kavirajk) [inline docs/sources/configuration/_index.md:3652]: @adeverteuil this should go into `index.template` file in the same directory. We generate `_index.md` from that template.

Comment (adeverteuil) [reply docs/sources/configuration/_index.md:3652]: Hi @kavirajk , I'm not sure what to do. All I see in `index.template` is a template variable like this:

```
### Supported contents and default values of `loki.yaml`

{{ .ConfigFile }}
```

I suppose I need to edit the content of `.ConfigFile`, but I don't know where to do that. I'm not familiar with the Loki code base, and I am guessing that this change needs to be done in a comment near the definition of this config parameter in the Loki code. Can a Loki developer please make this change where appropriate?

**Comment by JStickler** [comment]
Closing as already fixed by another PR.

**Pull request closed without merging**